app/models/move.rb

```diff
@@ -215,6 +215,10 @@ def important_events
     incident_events + (profile&.person_escort_record&.medical_events || [])
   end
 
+  def vehicle_registration
+    journeys.max_by(&:client_timestamp)&.vehicle&.dig('registration')
```

Comment:
not sure this will always return the correct answer... might be a good idea to exclude cancelled journeys?

Note also that the van (and thus registration) could change during the lifetime of the move. So the van coming to collect the person from the start might have a different registration to the van delivering the _same_ person to the destination